src/components/organisms/EntityDetails/ExecutionDetailsDrawer/ExecutionDetailsDrawer.tsx

```diff
@@ -131,7 +132,18 @@ const ExecutionDetailsDrawer: React.FC = () => {
             {selectedRow ? components[entity] : null}
           </ExecutionDetailsDrawerWrapper>
         </Drawer>
-      ) : null}
+      ) : (
+        <Drawer
+          bodyStyle={{display: 'flex', alignItems: 'center', justifyContent: 'center', background: Colors.slate800, fontSize: '48px'}}
+          headerStyle={{borderBottom: 0, padding: '40px 30px 0', backgroundColor: Colors.slate800}}
```

Comment:
Can we reuse these styles?

Reply:
Do you mean to `memo` them to avoid re-render?

Reply:
Ahh, I see, because it's exactly the same as in regular `Drawer` :) Will do.